product_price_history/models/product.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, api, fields
+
+
+class ProductPriceHistory(models.Model):
+    _inherit = 'product.price.history'
+
+    product_tmpl_id = fields.Many2one(
+        string='Product Template', store=True, related='product_id.product_tmpl_id')
+
+
+class ProductTemplate(models.Model):
```

Comment:
@eLBati Could you split => one file per model ?

Reply:
thanks, done